src/main/java/at/hannibal2/skyhanni/features/itemabilities/abilitycooldown/ItemAbilityCooldown.kt

```diff
@@ -42,6 +42,9 @@ class ItemAbilityCooldown {
             if (event.pitch == 0.61904764f && event.volume == 1f) {
                 ItemAbility.GYROKINETIC_WAND_LEFT.sound()
             }
+            if (event.pitch == 1f && event.volume == 1f){
+                ItemAbility.SHADOW_FURY.sound()
```

Comment:
Maybe check here if the player has a shadow fury in the hand (since the sound could be caused by different mechanics)